src/ReactWrapper.js

```diff
@@ -381,7 +381,10 @@ export default class ReactWrapper {
    * @returns {Object}
    */
   props() {
-    return this.single(n => getNode(n).props || {});
+    return this.single((n) => {
+      const node = getNode(n);
+      return node ? node.props || {} : {};
```

Comment:
i think in the Utils file there is a `propsOfNode` function that we should probably be using instead of this. It deals with things like null being passed in.